<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettingForGroundTargets</class>
 <widget class="QWidget" name="FormSettingForGroundTargets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Курсовой угол</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QDial" name="dialPsi">
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxPsi">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>99999.990000000005239</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>44</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>град</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>рад</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Скорость</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QDial" name="dialV">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxV">
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>500.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>39</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>м/с</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>км/ч</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>миль/ч</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Географические координаты</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelLon">
            <property name="text">
             <string>Долгота</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditLon">
              <property name="text">
               <string>000°00'00.00</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxLon">
              <property name="decimals">
               <number>12</number>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Широта</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEditLat">
              <property name="text">
               <string>000°00'00.00</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxLat">
              <property name="decimals">
               <number>12</number>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>X=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBoxX">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-999999999.999999046325684</double>
           </property>
           <property name="maximum">
            <double>999999999.999999046325684</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Z=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBoxZ">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-999999999.999999046325684</double>
           </property>
           <property name="maximum">
            <double>999999999.999999046325684</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Тип объекта</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxEnable">
       <property name="text">
        <string>Отображать в варианте целевой машины</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
